app/views/checkout/thankyou.scala.html

```diff
@@ -41,7 +44,7 @@
                 </p>
                 <p>Open times: Lines are open Monday&ndash;Friday 8.00am&ndash;5.30pm. Saturday &amp; Sunday 8.30am&ndash;12.30pm</p>
```

Comment:
Maybe we can just avoid switching and have two paragraphs here, one for UK the other for the rest of the word